src/components/Footer.tsx

```diff
@@ -1,15 +1,176 @@
-import React from 'react';
-import { Container } from 'styled-bootstrap-grid';
-import styled from 'styled-components';
+import React from "react";
+import { Container, Row, Col } from "styled-bootstrap-grid";
+import styled from "styled-components";
 
 const FooterContainer = styled.footer`
-  padding: 2rem 0;
-  margin-top: 10rem;
+  float: left;
+  width: 100%;
+  background: #eee;
+  font-size: 14px;
+  color: #8a8a8a;
+  padding: 30px 20px;
+`;
+
+const FooterTab = styled.div`
+  float: left;
+  width: 100%;
+  margin-bottom: 20px;
+`;
+
+const Title = styled.h4`
+  font-size: 20px;
+`;
+
+const List = styled.li`
+  margin: 5px 0;
+`;
+
+const Copyright = styled(Col)`
+  margin-left: auto;
+`;
+
+const Content = styled.p`
+  margin-bottom: 1.5rem;
 `;
 
 const Footer = () => (
   <FooterContainer>
-    <Container />
+    <Container>
+      <Row>
+        <Col lg={2} md={3} sm={6}>
+          <FooterTab>
+            <Title>
+              <span className="en">Navigate</span>
+            </Title>
+            <ul>
+              <List>
+                <a href="/pages/apps">
```

Comment:
This should be `Anchor` component. We don't want to use the default styles here.

Reply:
`Anchor` component? did you mean `Link` by NextJS ?

Reply:
`Anchor` for now since we dont have `/pages/apps` route setup